rust-toolchain.toml

```diff
@@ -18,6 +18,5 @@
 # Toolchain override for rustup
 
 [toolchain]
-channel = "nightly-2021-09-20"
-components = ["rust-src"]
+channel = "nightly"
```

Comment:
I would recommend to keep this pinned to "some" nightly to prevent things from breaking in a non-bisectable manner.

Reply:
Although I thought there are minimal feature dependency [1] we have for nightly releases but you are right we should pin that. BTW, I hope that nightly dependencies would be stabilised soon and we can switch to stable toolchain releases tracking mainline.

#![feature(c_size_t)] (https://github.com/rust-lang/rust/issues/88345)
#![feature(error_in_core)] (https://github.com/rust-lang/rust/issues/103765)